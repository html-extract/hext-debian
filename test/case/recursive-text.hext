<div id="target" @text:sentence/>
