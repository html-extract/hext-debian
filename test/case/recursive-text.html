<html>
<body>
<div id="target">
  So this
  <div>
    is
    <div>
      a
    </div>
    sentence
  </div>
  <div>
    all
    <div>
      over
    </div>
    the
    <span>
      place
    </span>
  </div>
</div>
</body>
</html>
