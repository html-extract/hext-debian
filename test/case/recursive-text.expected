{"sentence":"So this is a sentence all over the place"}
